### DISCUSSION TITLE
[3차 스프린트] 8월 1일 회의 계획

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#186] [백로그 논의]
#### 논의가 필요한 내용

안녕하세요 여러분!
오늘 회의에서 앞으로 해야 할 일에 대해 논의를 했습니다.
`8월 1일(목)` 회의 전에 미리 회의 안건을 정해두는 것이 좋을 것 같아서 이슈를 파게 됐습니다.

- [x] 기능 명세서 남은 부분 정리
- [x] 기능 명세서 우선 순위 정리
- [ ] 배포 방식 논의
    - (미확정) 프론트엔드 - S3 + CloudFront
- [ ] 2차 데모데이 배포 관련 롤백
    - 관련 내용 공유
- [ ] CI / CD 구조도 그리기

일단 오늘 나온 내용들을 위와 같이 적어두었는데
논의하고 싶은 사항이나 위 안건들 중에 나중에 논의해도 좋을 것 같다고 생각하는 부분을 코멘트로 적어주시면 감사하겠습니다!

참고) 물론 위 안건들을 내일 다 해결할 수 있을 것이라고 생각하진 않습니다 😅

**Comment by Parkhanyoung** [comment]
정리해주셔서 감사합니다 리브 👍🏻
